src/Amazon.S3/Model/S3MultipartUploadOperation_Internal.m

```diff
@@ -117,74 +117,8 @@ - (void)start
     
     self.error = nil;
     self.exception = nil;
-    
-    // Check if the upload was already started
-    S3ListMultipartUploadsRequest *listMPU = [[[S3ListMultipartUploadsRequest alloc] init] autorelease];
```

Comment:
We were never ever executing any code in `S3MultipartUploadOperation_Internal.m`, because of this check:
https://github.com/hudl/aws-sdk-ios-v1/blob/fe31755/src/Amazon.S3/S3TransferManager.m#L78
https://github.com/hudl/aws-sdk-ios-v1/blob/fe31755/src/Amazon.S3/S3TransferManager.m#L672-L679

https://github.com/hudl/modi/commit/89446c75f08297b239022964898b5a924cf78f1e
We bumped the multipart "chunking" threshold to 500 MB, meaning there was no way we could possibly attempt a multipart upload for any file.

Since this code has pretty much been dead for a long time, it can be safely removed.